### ISSUE TITLE
Bulk Extractor not installed on Bionic 18.04 AM distribution

### ISSUE THREAD

**Issue by peterVG** [archivematica/Issues#527]
**Expected behaviour**
Bulk Extractor is run on text files when "Examine Contents" option is selected during transfer.

**Current behaviour**
Examine Contents data is blank because Bulk Extractor is not installed.

**Your environment (version of Archivematica, OS version, etc)**

am19bionic.qa.archivematica.org

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by sevein** [comment]
CC @scollazo @mamedin

**Comment by peterVG** [comment]
![screen shot 2019-02-25 at 14 43 03](https://user-images.githubusercontent.com/672121/53373928-b542fb00-390b-11e9-9c7a-f3e5106bf0fa.png)

**Comment by sevein** [comment]
I don't have access to the QA machine but I've noticed that the package is installed but looks empty.

Same problem in 1.7 and 1.8 repos:
- https://packages.archivematica.org/1.7.x/ubuntu-externals/pool/main/b/bulk-extractor/
- https://packages.archivematica.org/1.8.x/ubuntu-externals/pool/main/b/bulk-extractor/

$ dpkg -L bulk-extractor
```
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/bulk-extractor
/usr/share/doc/bulk-extractor/changelog.Debian.gz
```

**Comment by ross-spencer** [comment]
Hi @sevein @mamedin would this affect the docker-compose environment too as I noticed a similar issue impacting bulk_extractor there when testing for Peter. I guess the follow up, is if so, will the fix cover that environment as well, or will we need to make a change there too?

**Comment by mamedin** [comment]
Hi @ross-spencer, for docker-compose environment we use the mcpclient-base image, and it uses the same ubuntu package:

https://github.com/artefactual/archivematica/blob/qa/1.x/src/MCPClient-base.Dockerfile#L36

**Comment by ross-spencer** [comment]
Awesome. Thanks @mamedin

**Comment by mamedin** [comment]
So the docker-image needs to be updated after publishing the new bionic package:

https://hub.docker.com/r/artefactual/archivematica-mcp-client-base/tags

**Comment by mamedin** [comment]
am-packbuild https://github.com/artefactual-labs/am-packbuild/pull/220  was merged, but when trying to add the new xenial package I found that the 1.9.x-extras repo uses `builk-extractor_1.5.5-1~14.04` (trusty) package, so I need to create a new PR to use the 1.5.5 version.

am-packbuild https://github.com/artefactual-labs/am-packbuild/pull/220 was using the 1.5.3 version because it was the latest release in github. The new PR will use the source tarball from the download page, that includes the 1.5.5 version.
